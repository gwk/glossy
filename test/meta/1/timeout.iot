{
  'cmd': '$PROJ/iotest.py -no-times test-meta/1/$NAME.iot',
  'code': 1,
}
