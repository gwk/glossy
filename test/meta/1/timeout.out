test-meta/1/timeout
test process timed out (1 sec) and was killed.
test command failed: `sleep 10`

================================================================


################################################################
TESTS FOUND: 1; BROKEN: 0; SKIPPED: 0; FAILED: 1.
